### PR TITLE
Add Clarendon filter

### PR DESCRIPTION
**Pull request by jagtalon**
😁

## Real:

img_4505
img_4507
img_4509

## CSS:

screen shot 2015-11-14 at 21 56 28
screen shot 2015-11-14 at 22 01 43
screen shot 2015-11-14 at 21 57 30

### CODE REVIEW

**Comment by una** [comment]
Thank you so much for these PRs!! It would be awesome if you added the filters to the *test/* page as well to see the side-by-side comparison!

Clarendon and Moon are missing from it :smile:

**Comment by jagtalon** [comment]
@una ready for testing!

File: source/scss/clarendon.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.3) saturate(1.2);
```
Comment (una) [inline source/scss/clarendon.scss]: I'd bring up the saturating and bring down the contrast a bit: `contrast(1.2) saturate(1.35)`

File: source/scss/clarendon.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  &:after {
+    background: rgba(239, 239, 239, .8);
+    mix-blend-mode: soft-light;
+  }
+
+  &:before {
+    background: rgba(17, 129, 147, .23);
```
Comment (una) [inline source/scss/clarendon.scss]: This is causing a bit of a blue wash -- what do you think of trying an opacity of `.1` instead of `.23`?

**Comment by jagtalon** [comment]
@una thanks I'll try those!

**Comment by jagtalon** [comment]
@una ready to go!

File: source/scss/clarendon.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.2) saturate(1.35);
```
Comment (una) [inline source/scss/clarendon.scss]: I was playing with this one and I think I got it super close!! Try `contrast(1.1) saturate(1.05);` for the filter

File: source/scss/clarendon.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  &:after {
```
Comment (una) [inline source/scss/clarendon.scss]: I would remove the `::after`

File: source/scss/clarendon.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    background: rgba(239, 239, 239, .8);
+    mix-blend-mode: soft-light;
+  }
+
+  &:before {
```
Comment (una) [inline source/scss/clarendon.scss]: Try this one: 

```
background: rgba(230, 245, 255, 0.4);
mix-blend-mode: overlay;
```

**Comment by jagtalon** [comment]
@una thank you for the thorough review! Will fix this PR and add more filters this weekend :wink:

File: source/scss/clarendon.scss (hunk 5 above)
Comment (jagtalon) [inline source/scss/clarendon.scss]: @una thanks this looks better! Although what do you think about `rgba(127, 187, 227, 0.2)`? Having a darker color seems more apt it seems. https://youtu.be/W-uspcrRe6Y

Comment (una) [inline source/scss/clarendon.scss]: YES! THAT!

**Comment by jagtalon** [comment]
Hey, @una. Moving to https://github.com/una/CSSgram/pull/137 since rebase got a little messy. :)

**Pull request closed without merging**